src/edu/washington/escience/myria/MyriaSystemConfigKeys.java

```diff
@@ -57,98 +56,76 @@ private MyriaSystemConfigKeys() {
    * */
   public static final String TCP_CONNECTION_TIMEOUT_MILLIS = "tcp.connection.timeout.milliseconds";
```

Comment:
I guess if you're gonna replace the Java-style dotted names with underscores (unless there's implicit hierarchy), you should do it everywhere.